app/ledger.js

```diff
@@ -286,6 +286,25 @@ if (ipc) {
     if (balanceTimeoutId) clearTimeout(balanceTimeoutId)
     balanceTimeoutId = setTimeout(getBalance, 5 * msecs.second)
   })
+
+  ipc.on(messages.OPEN_LEDGER_TRANSACTION_CSV, (event, viewingIds, csvFilename) => {
+    if (client) {
+      var txCsvText = client._getTransactionCSVText(viewingIds)
```

Comment:
i would replace all of the below with `win.webContents.downloadURL(url)`, where `url` is `txCsvText` as a Blob object URL or perhaps a data URL. then the CSV file will be automatically written to the default downloads folder, and the user can click on the downloads bar to open it.